cmd/aries-agentd/main_test.go

```diff
@@ -103,7 +103,7 @@ func validateRequests(t *testing.T) {
 								"@type": "did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/didexchange/1.0/invitation"
 							}`),
 				"application/didcomm-envelope-enc"),
-			expectedStatus:     http.StatusAccepted,
+			expectedStatus:     http.StatusInternalServerError,
```

Comment:
Is this the status for the protocol doesn't yet have action callback listeners?

There seems to be a couple of relevant error codes to choose from: https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#5xx_Server_errors

e.g., 503

Reply:
@troyronda yes. The error codes are set in [inbound.go](https://github.com/hyperledger/aries-framework-go/blob/master/pkg/didcomm/transport/http/inbound.go#L62).

Reply:
Ah. We should consider a TODO where we have a more specific code like 503.

Reply:
#271